docs/conceptual/More-about-how-ROCm-uses-PCIe-Atomics.rst

```diff
@@ -1,156 +1,92 @@
 .. meta::
    :description: How ROCm uses PCIe atomics
-   :keywords: PCIe, PCIe atomics, atomics, BAR memory, AMD, ROCm
+   :keywords: PCIe, PCIe atomics, atomics, AMD, ROCm
 
 *****************************************************************************
-How ROCm uses PCIe atomics
+How ROCm Uses PCIe Atomics
```

Comment:
https://rocm.docs.amd.com/en/latest/contribute/contributing.html#editing-and-adding-to-the-documentation

https://developers.google.com/style/headings

Please use sentence case @prbasyal-amd